apps/studio/components/interfaces/Settings/General/CustomDomainConfig/CustomDomainVerify.tsx

```diff
@@ -31,18 +33,26 @@ const CustomDomainVerify = ({ projectRef, customDomain, settings }: CustomDomain
         if (res.status === '2_initiated') setIsNotVerifiedYet(true)
       },
     })
+
   const { mutate: deleteCustomDomain, isLoading: isDeleting } = useCustomDomainDeleteMutation()
 
   const hasCAAErrors = customDomain.ssl.validation_errors?.reduce(
     (acc, error) => acc || error.message.includes('caa_error'),
     false
   )
+  const isValidating = (customDomain.ssl.txt_name ?? '') === ''
 
   const onReverifyCustomDomain = () => {
     if (!projectRef) return console.error('Project ref is required')
     reverifyCustomDomain({ projectRef })
   }
 
+  useInterval(
+    onReverifyCustomDomain,
```

Comment:
we can just set a refetchInterval for this - i don't think we need a separate hook?